### ISSUE TITLE
Add post-install building to plugins

### ISSUE THREAD

**Issue by pfrazee** [pfrazee/beaker#59]
The native modules in the plugins have to be rebuilt for the current Electron version, due to some ABI hijinks. This needs to happen automatically, now, after install or update, to work with Beaker's plugin installer.

**Comment by pfrazee** [comment]
There's an upstream bug that makes dat (the plugin that currently needs this) hard to work on: https://github.com/mafintosh/utp-native/issues/7

**Comment by pfrazee** [comment]
I was discovering a lot of difficult edge-cases in the in-browser plugin installer, which made me worry that, without proper infrastructure to support this complex of a feature, the product would be be buggy. Building Electron for multiple platforms is already pretty difficult; making a more complex packaging system, like this, would be much worse.

The feedback I've received from other protocol designers is somewhat split. While I think some protocol devs would strongly appreciate having the in-browser plugin installer, there are others who are more likely to need a custom packaging or fork of beaker. Therefore, since protocol engineers are, ultimately, a minority of Beaker's intended users, and, among them, the need for an installer is divided, I've decided not to make this a priority.

Docs have been updated to reflect the new solution: plugins are installed locally, and Beaker must be rebuilt from source to enable them.
